tests/cypress/component/llamalend/pegkeeper.cy.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { PegKeeper } from '@/loan/components/PagePegKeepers/components/PegKeeper'
+import { PEG_KEEPERS } from '@/loan/components/PagePegKeepers/constants'
+import { ClientWrapper, type Config } from '@/support/helpers/ClientWrapper'
+import { createTestWagmiConfigFromVNet, withVirtualTestnet } from '@/support/helpers/vnet'
+
+const getVnet = withVirtualTestnet((uuid) => ({
+  slug: `pegkeepers-${uuid}`,
+  display_name: `Pegkeepers (${uuid})`,
+  fork_config: {
+    block_number: '23039344',
+  },
+}))
+
+const TestComponent = ({ config }: { config: Config }) => (
+  <ClientWrapper config={config} autoConnect={false}>
+    <PegKeeper {...PEG_KEEPERS[0]} sx={{ maxWidth: '30rem' }} testId="pegkeeper" />
+  </ClientWrapper>
+)
+
+describe('PegKeeper', () => {
```

Comment:
This should be in the `loan` folder not `llamalend` also should it not be called "peg stability reserves"?

Reply:
At one point we renamed pegkeepers to peg stability reserves and I didn't want a giant PR renaming every single file, so there's some intermingling. Renamed it regardless

Technically it's not loan specific but it does live in the loan app because that's where it originated. Still looking forward to merge both apps in a single llamalend app 😅